aruco_cam:
  ros__parameters:
    width: 320
    height: 240
    marker_shape: "DICT_4X4_50"
    cam_matrix: [265.47975665, 0.0, 155.7742281, 0.0, 265.69425432, 116.04458026, 0.0, 0.0, 1.0]
    dist_coeff: [1.26778002e-01, -7.16508224e-04, -3.06511403e-03, 5.77083117e-03, -9.74301238e-01]

# domain_id : 213
# camMatrix
# [256.61118671, 0.0, 161.40241153, 0.0, 257.21410209, 118.41154184, 0.0, 0.0, 1.0]
# distCoef
# [1.28394909e-01, 7.42384842e-02, 7.25671420e-04, 1.43449039e-03, -1.06879001e+00]
# ------
# domain_id : 214
# camMatrix
# [249.26653787, 0.0, 160.31833382, 0.0, 250.05718578, 114.24750264, 0.0, 0.0, 1.0]
# distCoef
# [1.66805069e-01, -3.68234503e-01, 4.76781203e-05, 2.62091227e-03, 1.72290440e-01]
# ------
# domain_id : 215
# camMatrix
# [265.47975665, 0.0, 155.7742281, 0.0, 265.69425432, 116.04458026, 0.0, 0.0, 1.0]
# distCoef
# [1.26778002e-01, -7.16508224e-04, -3.06511403e-03, 5.77083117e-03, -9.74301238e-01]

